app/js/services/cp_analytics_api_client.js

```diff
@@ -3,3 +3,15 @@ import { get, post, parseJSON } from './auth'
 
 export const getTableauToken = () => get(`${CP_ANALYTICS_API}tableau/token`).then(parseJSON)
 export const postStats = (type, key, value) => post(`${CP_ANALYTICS_API}stats`, { type, key, value })
+export const loginToSentisum = () => {
+  let newTab = window.open('', '', '')
+
+  return get(`${CP_ANALYTICS_API}sentisum/login`)
+    .then(parseJSON)
+    .then(json => { newTab.location = json.url })
+    .catch((e) => {
+      newTab.close()
+      window.alert('Login failed')
+      window.console.error(e)
```

Comment:
I hope we will not crash here due to browser not supporting object *console*.

Reply:
You are right, I will wrap it to `if (window.console) {}`